pypuppetdb/__init__.py

```diff
@@ -121,13 +119,8 @@ def connect(api_version=3, host='localhost', port=8080, ssl_verify=False,
 
     :raises: :class:`~pypuppetdb.errors.UnsupportedVersionError`
     """
-    if api_version == 3:
-        return v3.API(host=host, port=port,
-                      timeout=timeout, ssl_verify=ssl_verify, ssl_key=ssl_key,
-                      ssl_cert=ssl_cert, protocol=protocol, url_path=url_path,
-                      username=username, password=password)
-    if api_version == 2:
-        return v2.API(host=host, port=port,
+    if api_version == 4:
```

Comment:
In that case we might as well remove the `api_version` parameter globally since it will no longer be used. I'm not sure what your position on backward-compatibility is in this case. Thoughts?